File: utils/IplDataScrapper/IplDataScrapper/Player.cs
namespace IplDataScrapper
{
    class Player
    {
        public string TeamName { get; set; }
        public string Name { get; set; }
        public string Role { get; set; }
        public double AuctionPrice { get; set; }
        public string Nation { get; set; }

        public Player(string teamName, string name, string role, double auctionPrice, string nation)
        {
            TeamName = teamName;
            Name = name;
            Role = role;
            AuctionPrice = auctionPrice;
            Nation = nation;
        }
    }
}

File: ipl-tournament/src/com/tarktech/training/ipl/domain/Player.java
package com.tarktech.training.ipl.domain;

public class Player {
    private String name;
    private PlayerRole Role;
    private int jerseyNo;

    public Player(String name, PlayerRole Role, int jerseyNo) {
        this.name = name;
        this.Role = Role;
        this.jerseyNo = jerseyNo;
    }

    public PlayerRole getRole() {
        return Role;
    }

    @Override
    public String toString() {
        return name;
    }

    public String getName() {
        return name;
    }

    public int getJerseyNo() {
        return jerseyNo;
    }
}